src/main/scala/org/broadinstitute/dsde/firecloud/service/EntityService.scala

```diff
@@ -72,7 +72,7 @@ trait EntityService extends HttpService with PerRequestCreator with FireCloudDir
                   pathPrefix(Segment) { entityName =>
                     pathEnd {
                       requireUserInfo() { _ =>
-                        passthrough(entityTypeUrl + "/" + entityName, HttpMethods.GET, HttpMethods.PATCH, HttpMethods.DELETE)
+                        passthrough(entityTypeUrl + "/" + entityName, HttpMethods.GET, HttpMethods.PATCH)
```

Comment:
the `DELETE` path is not documented in orch swagger, and does not exist in Rawls ... i.e. it's undocumented and it doesn't work. So I'm deleting it.